server/application-server/src/main/java/de/tum/in/www1/hephaestus/chat/session/SessionController.java

```diff
@@ -0,0 +1,35 @@
+package de.tum.in.www1.hephaestus.chat.session;
+
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import java.util.List;
+
+import org.springframework.http.ResponseEntity;
+
+@RestController
+@RequestMapping("/sessions")
```

Comment:
Our usual convention is to use the singular in endpoint naming. So `/session`.

Same goes for the `MessageController`.